functoria installation fails when ounit.1.1.2 is installed

Fails with:

```
# File "tests/test_functoria_command_line.ml", line 17, characters 5-11:
# Error: Unbound module OUnit2
# Command exited with code 2.
# Makefile:7: recipe for target 'build' failed
```

Not 100% sure but I think we need to add this declaration in functoria opam file:

```
depopts: ["ounit" {>= 2.0.0}]
```

--- a/packages/functoria/functoria.1.0.0/opam
+++ b/packages/functoria/functoria.1.0.0/opam
@@ -25,9 +25,6 @@ depends: [
   "rresult"
   "fmt"
   "ocamlgraph"
-  "ounit" {test}
+  "ounit" {test & >="2.0.0"}
 ]
 available: [ocaml-version >= "4.01" & ocaml-version < "4.03"]
-conflicts: [
-  "ounit" {< "2.0.0"}
-]

--- a/packages/functoria/functoria.1.1.0/opam
+++ b/packages/functoria/functoria.1.1.0/opam
@@ -25,6 +25,6 @@ depends: [
   "rresult"
   "fmt"
   "ocamlgraph"
-  "ounit" {test}
+  "ounit" {test & >="2.0.0"}
 ]
 available: [ocaml-version >= "4.01"]

--- a/packages/functoria/functoria.1.1.1/opam
+++ b/packages/functoria/functoria.1.1.1/opam
@@ -27,6 +27,6 @@ depends: [
   "rresult"
   "fmt"
   "ocamlgraph"
-  "ounit" {test}
+  "ounit" {test & >="2.0.0"}
 ]
 available: [ocaml-version >= "4.01"]

--- a/packages/functoria/functoria.2.0.0/opam
+++ b/packages/functoria/functoria.2.0.0/opam
@@ -30,6 +30,6 @@ depends: [
   "logs"
   "bos"
   "fpath"
-  "ounit" {test}
+  "ounit" {test & >="2.0.0"}
 ]
 available: [ocaml-version >= "4.03"]

--- a/packages/functoria/functoria.2.0.1/opam
+++ b/packages/functoria/functoria.2.0.1/opam
@@ -30,6 +30,6 @@ depends: [
   "logs"
   "bos"
   "fpath"
-  "ounit" {test}
+  "ounit" {test & >="2.0.0"}
 ]
 available: [ocaml-version >= "4.03"]

